CHANGELOG.rst

```diff
@@ -8,6 +8,10 @@ Fixed:
 ^^^^^^
 - Heatmap values to be converted from N to µN before sending to FE.
 
+Changed:
+^^^^^^^^
+- Limited logging of /system_status to only when status code is not 200
```

Comment:
This is too technical for our users. It should be something higher level that just mentions that unnecessary log lines have been remove from log files

Reply:
changed to 

```
Removed:
^^^^^^^^
- Unnecessary logging from log files
```

Reply:
We don't use a `Removed` subsection, we have `Added`, `Changed`, and `Fixed`. If you look through the changelog I believe all removals go under `Changed`